sdks/go/pkg/beam/runners/dataflow/dataflow.go

```diff
@@ -59,6 +59,7 @@ var (
 	numWorkers           = flag.Int64("num_workers", 0, "Number of workers (optional).")
 	maxNumWorkers        = flag.Int64("max_num_workers", 0, "Maximum number of workers during scaling (optional).")
 	diskSizeGb           = flag.Int64("disk_size_gb", 0, "Size of root disk for VMs, in GB (optional).")
+	diskType             = flag.String("disk_type", "", "Type of root disk for VMs (optional).")
```

Comment:
TIL that the common flag for this from the python SDK is apparently "worker_disk_type". Technically Python has it aliased to both that and --disk_type, so we may want to do the same (this would require a flag.StringVar call).

https://github.com/apache/beam/blob/aaab98a2927669875ee8ab6fbd098e02c516eab2/sdks/python/apache_beam/options/pipeline_options.py#L833

Reply:
Would that just be adding a `--worker_disk_type` flag that stores its value in the `--disk_type` flag's variable? How would that interact when considering defaults?